mailpoet/lib/EmailEditor/Integrations/Core/Renderer/Blocks/ListBlock.php

```diff
@@ -0,0 +1,42 @@
+<?php declare(strict_types = 1);
+
+namespace MailPoet\EmailEditor\Integrations\Core\Renderer\Blocks;
+
+use MailPoet\EmailEditor\Engine\Renderer\BlockRenderer;
+use MailPoet\EmailEditor\Engine\SettingsController;
+
+// We have to avoid using keyword `List`
+class ListBlock implements BlockRenderer {
+  public function render($blockContent, array $parsedBlock, SettingsController $settingsController): string {
+    $contentStyles = $settingsController->getEmailContentStyles();
+    return str_replace('{list_content}', $blockContent, $this->prepareColumnTemplate($parsedBlock, $contentStyles));
+  }
+
+  private function prepareColumnTemplate(array $parsedBlock, array $contentStyles): string {
```

Comment:
I think we should come up with a better name, `prepareColumnTemplate` maybe something like `getBlockWrapper` or `getStyleResetWrapper` or something else. What do you think?